lib/chef/web/core/helpers.rb

```diff
@@ -0,0 +1,34 @@
+require 'chef/web/core'
+require 'chef/web/core/component'
+require 'nokogiri'
+
+class Chef
+  module Web
+    module Core
+      module Helpers
```

Comment:
We have `URLHelpers`. Pehaps this should be something more specific, like `AssetHelpers`. Maybe. This is fine too.